src/mono/browser/runtime/runtime.c

```diff
@@ -188,6 +188,33 @@ init_icall_table (void)
 #endif
 }
 
+// Keep synced with FixupSymbolName from src/tasks/Common/Utils.cs
+static void fixupSymbolName(char *key, char *fixedName) {
```

Comment:
I see that this method being copied to multiple places, which makes me wonder if it could be added to a location where is accessible to all the places that needs to be called. One place I could think of is `src/mono/mono/eglib/gstring.c` or `gstr.c`.

Reply:
Moved to `src/mono/mono/metadata/native-library.c`.